nft_ingester/src/backfiller.rs

```diff
@@ -627,7 +628,7 @@ impl<'a, T: Messenger> Backfiller<'a, T> {
         &mut self,
         btree: &BackfillTree,
     ) -> Result<Option<i64>, IngesterError> {
-        let address = Pubkey::new(btree.unique_tree.tree.as_slice());
+        let address = Pubkey::try_from(btree.unique_tree.tree.as_slice()).expect("valid pubkey");
```

Comment:
Previously with `Pubkey::new` we also had killed process, `Pubkey::try_from().expect()` is equal to the old code. I think it's ok to leave `.expect()`.

Reply:
I guess we can leave it as-is although I would prefer for it to return an error instead of panic.  But it is not a blocker as its current behavior today.

Reply:
fixed